distribution/helm/vector/README.md

```diff
@@ -0,0 +1,59 @@
+# Vector
+
+[Vector](https://vector.dev) is used to ship Kubernetes and host logs to multiple outputs.
+
+## Prerequisites
+
+- Kubernetes 1.10+
+
+## Note
+
+By default this chart only ships a single output to container logs.
```

Comment:
I'm not clear what this means?

Reply:
It probably means that the default `vector.toml` configuration has a single sink, a `console` sink, to which container logs are outputted.